docs/visualization/visualize_lfp_responses.ipynb

```diff
@@ -84,7 +84,7 @@
      "name": "stderr",
```

Comment:
<p>maybe make it "selected stimulus" instead of just "stimulus" to be extra clear we're talking about specific stim</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>